### ISSUE TITLE
Support for some host.json config at the function level

### ISSUE THREAD

**Issue by christopheranderson** [Azure/azure-webjobs-sdk-script#1684]
Today, settings like the queue visibility timeout is done at the host level, not the Function level, which means if I have two functions that need different settings, I have to use different Function Apps. It would be nice if I could over write the host.json settings with function specific settings, where possible.

From UserVoice: https://feedback.azure.com/forums/355860/suggestions/19103767

**Comment by paulbatum** [comment]
We should evaluate whether this feasible in the context of webjobs 3.x.

**Comment by mathewc** [comment]
Yes, this would be great to add sooner rather than later. We can do it now for queues, building on the core SDK custom queue processor factory APIs. We might also consider adding top level knobs to the attribute itself.

**Comment by BowserKingKoopa** [comment]
This would be great. Having these things defined globally instead of at the function level is bizarre.

**Comment by ogrygorchuk** [comment]
Guys, any updates on that? 
Anyone have found possibility to set different values for different queues?
That`s actually where I stuck now...

**Comment by paulbatum** [comment]
Currently we do not have an ETA for this feature. The best approach you can use right now is to deploy separate function apps.

**Comment by ogrygorchuk** [comment]
Thank you for answer.
